_pages/business-units/18f/chapters/account-manager.md

```diff
@@ -81,10 +81,26 @@ If you’re worried about your project, bring up any concerns or blockers with y
 
 ## Resources for Account Managers
 
+### General
 - [Custom Partnerships Tracker Trello board](https://trello.com/b/kZ7PUggv/custom-partnerships-tracker)
 - [Project resources Drive folder](https://drive.google.com/drive/u/0/folders/1L9qqS6-b-emvlWJ4JPCG58LW62bbV361)
-- [Cost estimator templates](https://docs.google.com/spreadsheets/d/11yeM0k6bLKVmvkmN2JM-a26a0S-7QwSwImM7saaV1EY/edit#)
-- [SOW templates](https://drive.google.com/drive/u/0/folders/1tDBr9Evh4ZGPqlJUxirlzyGtnoXtf6wS)
 - [IAA/Appropriations law class materials](https://docs.google.com/document/d/1HHWaKrpzFtypk4Ymg7ZQAgpyfw4-U_1YwdWsYC74O_c/edit)
 - [Design your weekly sync meetings](https://docs.google.com/document/d/1grOSdOcuvd9O6yNIykCGNwk2Dxs10GXR8hAU_-uET1I/edit)
 
+### Business Development
+- [SME calls](https://docs.google.com/document/d/1vP4H9X1jMN0feuNHrw_vuAwtxBGQX7R56-sRl3UGFQk)  
+- [Proposal writing](https://docs.google.com/document/d/1U-TTrYTFW-UYkgs_gdhPxifi9ExW-9NyTCG-AfbO-6I) 
+- [SOW templates](https://drive.google.com/drive/u/0/folders/1tDBr9Evh4ZGPqlJUxirlzyGtnoXtf6wS) 
+- [Cost estimator templates](https://docs.google.com/spreadsheets/d/11yeM0k6bLKVmvkmN2JM-a26a0S-7QwSwImM7saaV1EY) 
+- [Generating follow-on work](https://docs.google.com/document/d/1cK5lapUzuf4KX4VTVpV22TvRjCHH86phoLiyXzFO3Ss)  
+
+### Pre-flight 
+- [Partner Meeting Agenda and Notes](https://docs.google.com/document/d/1D_MZ7MTGjbifO_H9zb8-kK8TBgk0-Z76s7xDW7SO9y8/edit#)   
+- [SOW talking points for AMs](https://docs.google.com/document/d/1wQZZybx3bgRRs3bsDCeSqC6LTKRqNufuqQZlGCqoa9I/edit) 
+
+### Staffing requests
+- [How staffing works](https://github.com/18F/staffing) 
+- [Staffing skills decoder](https://docs.google.com/document/d/1PzQGM1QrJ6bzIkzKUDd2oUWks3_9_j-jjbvhzdrTOuY/edit?folder=14Et2Gni7q8VhqsKljbMQHtMFlnxlE80d#) 
+- [Choosing a project lead](https://handbook.18f.gov/leading-projects/#how-do-we-decide-who-leads-each-project)  
```

Comment:
Ideally, we should use baseurl here rather than the domain, especially since we've moved to TTS.gov. Not a blocker, though

Reply:
Has this been resolved?

Reply:
@tbaxter-18f I don't know what you mean. 🙊 Can you elaborate?

Reply:
Sure! On most pages in the handbook, in the source you'll see links that look like this:
`[training]({{site.baseurl}}/olu/)`
That that `{{site.baseurl}}` bit is doing is resolving the site domain for you, so that you don't have to write in `handbook.tts.gov`. This makes it more resilient if the handbook moves (like it just did) and also makes it easier to test the links.

Reply:
ah derp that was either than i realized. Cool

Reply:
so this is good to go, yeah?